aws/logs_monitoring/lambda_function.py

```diff
@@ -635,7 +636,13 @@ def extract_trace_payload(event):
 def extract_metric(event):
     """Extract metric from an event if possible"""
     try:
-        metric = json.loads(event["message"])
+        message = event["message"]
+        if DD_LEGACY_MONITORING_LINE and message.strip().startswith("MONITORING|"):
+            metric = extract_legacy_metric(message)
+            if metric:
+                return metric
```

Comment:
perhaps we should also add `metric["t"] += event[DD_CUSTOM_TAGS].split(",")` before return?